source/update.rst

```diff
@@ -210,3 +210,47 @@ Percona's Version Service:
        versionServiceEndpoint: https://check.percona.com
        schedule: "0 0 * * *"
    ...
+
+.. _operator-update-smartupdates-major:
+
+Percona Server for MongoDB major version upgrades
+*************************************************
+
+Normally automatic upgrade takes place within minor versions (for example,
+from ``4.2.11-12`` to ``4.2.12-13``) of MongoDB. Major versions upgrade (for
+example moving from ``4.2-recommended`` to ``4.4-recommended``) is more
+complicated task which might potentially affect how data is stored and how
+applications interacts with the database (in case of some API changes). 
+
+Such upgrade is supported by the Operator within one major version at a time:
+for example, to change Percona Server for MongoDB major version from 4.0 to 4.4,
+you should first upgrade it to 4.2, and later make a separate upgrade from 4.2
+to 4.4. The same is true for major version downgrades.
+
+.. note:: It is recommended to take a backup before upgrade, as well as to
+   perform upgrade on staging environment.
+
+Major version upgrade can be initiated using the :ref:`upgradeoptions-apply`
+key in the ``deploy/cr.yaml`` configuration file:
+
+.. code:: yaml
+
+   spec:
+     upgradeOptions:
+       apply: 4.4-recommended
+
+.. note:: When making downgrades (e.g. changing version from 4.4 to 4.2), make
+   sure to remove incompatible features that are persisted and/or update
+   incompatible configuration settings. Compatibility issues between major
+   MongoDB versions can be found in `upstream documentation <https://docs.mongodb.com/manual/release-notes/4.4-downgrade-standalone/#prerequisites>`_.
+
+By default the Operator uses `FeatureCompatibilityVersion (FCV) <https://docs.mongodb.com/manual/reference/command/setFeatureCompatibilityVersion/>`_
+to make sure backwards-incompatible features are not not automatically enabled 
+with the major version upgrade (which is recommended and safe behavior).
```

Comment:
Would be great to expand here, that setting FCV is NOT recommended right after binaries are upgraded. It is recommended to upgrade the binaries by setting apply: flag, then waiting for few hours and check if everything is working and only after that set the setFCV flag to true. 
Setting setFCV flag to true with apply: flag at the same time should be done only if the whole procedure is tested on staging and the user is 100% sure about it. 

It is important, cause for now we do not support automated rollback if FCV is set (which we mention below).